### DISCUSSION TITLE
Stereo Delay

### DISCUSSION THREAD

**Discussion by betonizm** [SynthstromAudible/DelugeFirmware#2013] [Ideas]
Why not make the stereo delay not just Enabled/Disabled, but an adjustable bipolar parameter, so that you can change the pan direction of the delay and the width of the stereo field.

**Comment by duncanrmi** [comment]
I recently acquired a torso S4 sample/granny-synth thing, & while it's not without its issues in the first release of firmware, the stereo delay implementation is very interesting, possibly unique.
the width control, away from a central mono position, does the usual thing when turned cw, of expanding the ping-pong panning. however, when turned ccw, it seems to spread the left & right repeats using some sort of scattering algorithm, perhaps borrowed from the way the 'spray' function works in granular synthesis.
a quick/dirty way to do this would be a random modulation of one or both of the delay times, almost like a chorus effect.
